### PR TITLE
Add datastore to permanently store entities

### PR DESCRIPTION
**Pull request by AymarMuhikira**
-add maven dependency to poxml to be able to reference DataStore
-modify DataServlet doGet function to store comments in DataStore

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -51,10 +54,18 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
 
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-      num_cmnts++;
-      String Cmt = request.getParameter("cmnt");
-      comments.add("#Comment"+num_cmnts+": "+Cmt);
-      response.sendRedirect("/index.html");
+    num_cmnts++;
```
Comment (gbuenoandrade) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Nit: Local variable names should be written in lowerCamelCase (https://google.github.io/styleguide/javaguide.html#s5.2.7-local-variable-names)

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -51,10 +54,18 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
...
+    String Cmt = request.getParameter("cmnt");
```
Comment (gbuenoandrade) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Nit: Local variable names should be written in lowerCamelCase (https://google.github.io/styleguide/javaguide.html#s5.2.7-local-variable-names)

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 3]
```diff
@@ -51,10 +54,18 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
...
+    comments.add("#Comment"+num_cmnts+": "+Cmt);
```
Comment (gbuenoandrade) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Nit: Missing space

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 4]
```diff
@@ -51,10 +54,18 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
...
+
+    Entity taskEntity = new Entity("Comment");
+    taskEntity.setProperty("Number", num_cmnts);
+    taskEntity.setProperty("text", comments.get(num_cmnts-1));
```
Comment (gbuenoandrade) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Nit: Use the same case convention for all properties

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (AymarMuhikira) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: fixed the issue

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 2 above)
Comment (AymarMuhikira) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Thank you, fixed the issue

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 4 above)
Comment (AymarMuhikira) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: changed number to lower case

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 3 above)
Comment (AymarMuhikira) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java]: Thank you, changed the style

**Review by gbuenoandrade — APPROVED** [review]
Awesome, looks great now! Good job :)

LGTM

**Pull request merged**